README.md

```diff
@@ -248,8 +248,15 @@ QAndQDotBounds waits for a biorbd model and returns a structure with the minimal
 How convenient!
 ```python
 x_bounds = QAndQDotBounds(biorbd_model)
-```
-Then, override the first and last column to be 0, that is the sideways and rotation to be null for both the position and the velocities
+
+
+The first element of x_bounds is the degree of freedom declared in order in the bio.Mod file.
+In this case, the degree of freedom with index 0 is translation y, and the index 1 refers to rotation x. 
+Finally, the index 2 and 3 are respectively the velocity of translation y and rotation x 
+
+The time is discretized in nodes wich is the second element declared in x_bounds
```

Comment:
what are you calling "element"? I though you were referring to a subset of the rows (as the full rows are not just *q* based as you seem to imply here: `The first element of x_bounds is the degree of freedom declared in order in the bio.Mod file.`, but to the full path constraints of the problem (including *qdot*). But now, `element` seems to refer to dimensions (i.e. the columns), because you refer to time. If so, please use `dimension` and not element. Also, make sure you don't mislead the user by saying the rows refer to *q* solely as it is not. 

Finally, the columns in x_bounds can be discretized the same way as the problem, but this is not even the default option. The default is 3 columns, that is the  `starting node`, the `intermediate nodes` and the `final node`

Reply:
mot "element" remplacé par "dimension". Nous voulions parler de dimension en effet pour expliquer que les premières valeurs rentrées dans les crochets de x_bounds étaient les degrés de liberté q et leur vitesse qdot puis les noeuds.